assets/js/components/PremiumPill.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function PremiumPill() {
+  return (
+    <p className="ml-1 px-2 inline-flex text-xs leading-5 font-semibold rounded-full bg-green-100 text-green-800">
+      Premium
+    </p>
```

Comment:
I'm noticing that this markup is pretty much identical to the one used to declare the check ID, maybe we could have a component for both purposes?

Reply:
Hey @dottorblaster ,
I have modified the `PremiumPill` to use the default `Pill` component.
I modified the `Pill` component to accept a `size` prop, which is not nothing more than more options to add classnames.
I didn't want to have a hardcoded map, as sizing is always really cusotmizable.
What do you think?

Reply:
I would prefer the hardcoded map because paddings and sizes should be a little bit opinionated, but your call :+1:

Reply:
Ok. I will go with the map. If we need further customization we will have a look at that point